kuksa-val-server/test/unit-test/readme.md

```diff
@@ -2,8 +2,6 @@
 
 ### Requires Boost version > 1.59
 
-Copy the vss signal tree json manually from [here](https://github.com/GENIVI/vehicle_signal_specification/blob/master/vss_rel_1.0.json) to the build directory once you have build the unit test binary using the following instaructions.
```

Comment:
This text seems to be obsolete. Which json file the unit tests use is hard coded and the used json file already exists in the repo.